app/models/early_career_payments/eligibility.rb

```diff
@@ -217,132 +340,8 @@ def award_amounts
-        AwardAmount.new(
-          itt_subject: :chemistry,
-          itt_academic_year: AcademicYear.new(2020),
-          claim_academic_year: AcademicYear.new(2022),
-          base_amount: 2_000,
-          uplift_amount: 3_000
-        ),
-        AwardAmount.new(
-          itt_subject: :foreign_languages,
-          itt_academic_year: AcademicYear.new(2020),
-          claim_academic_year: AcademicYear.new(2022),
-          base_amount: 2_000,
-          uplift_amount: 3_000
-        ),
-        AwardAmount.new(
-          itt_subject: :mathematics,
-          itt_academic_year: AcademicYear.new(2018),
-          claim_academic_year: AcademicYear.new(2023),
-          base_amount: 5_000,
-          uplift_amount: 7_500
-        ),
-        AwardAmount.new(
-          itt_subject: :mathematics,
-          itt_academic_year: AcademicYear.new(2020),
-          claim_academic_year: AcademicYear.new(2023),
-          base_amount: 2_000,
-          uplift_amount: 3_000
-        ),
-        AwardAmount.new(
-          itt_subject: :physics,
-          itt_academic_year: AcademicYear.new(2020),
-          claim_academic_year: AcademicYear.new(2023),
-          base_amount: 2_000,
-          uplift_amount: 3_000
-        ),
-        AwardAmount.new(
-          itt_subject: :chemistry,
-          itt_academic_year: AcademicYear.new(2020),
-          claim_academic_year: AcademicYear.new(2023),
-          base_amount: 2_000,
-          uplift_amount: 3_000
-        ),
-        AwardAmount.new(
-          itt_subject: :foreign_languages,
-          itt_academic_year: AcademicYear.new(2020),
-          claim_academic_year: AcademicYear.new(2023),
-          base_amount: 2_000,
-          uplift_amount: 3_000
-        ),
-        AwardAmount.new(
-          itt_subject: :mathematics,
-          itt_academic_year: AcademicYear.new(2019),
-          claim_academic_year: AcademicYear.new(2024),
-          base_amount: 5_000,
-          uplift_amount: 7_500
-        ),
-        AwardAmount.new(
-          itt_subject: :mathematics,
-          itt_academic_year: AcademicYear.new(2020),
-          claim_academic_year: AcademicYear.new(2024),
-          base_amount: 2_000,
-          uplift_amount: 3_000
-        ),
-        AwardAmount.new(
-          itt_subject: :physics,
-          itt_academic_year: AcademicYear.new(2020),
-          claim_academic_year: AcademicYear.new(2024),
-          base_amount: 2_000,
-          uplift_amount: 3_000
-        ),
-        AwardAmount.new(
-          itt_subject: :chemistry,
-          itt_academic_year: AcademicYear.new(2020),
-          claim_academic_year: AcademicYear.new(2024),
-          base_amount: 2_000,
-          uplift_amount: 3_000
-        ),
-        AwardAmount.new(
-          itt_subject: :foreign_languages,
-          itt_academic_year: AcademicYear.new(2020),
-          claim_academic_year: AcademicYear.new(2024),
-          base_amount: 2_000,
-          uplift_amount: 3_000
-        )
-      ]
```

Comment:
@nomasprime Good to have refactored but I feel a step has been missed by not refactoring into its own class / concern